zag/persistence/backends/sqlalchemy/alembic/versions/ad1e87d0f747_adding_atomdetails_flowdetails_parent_.py

```diff
@@ -0,0 +1,25 @@
+"""adding_atomdetails_flowdetails_parent_uuid_index
+
+Revision ID: ad1e87d0f747
+Revises: 2ad4984f2864
+Create Date: 2019-05-09 12:35:07.004172
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'ad1e87d0f747'
+down_revision = '2ad4984f2864'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_index(op.f('ix_flowdetails_parent_uuid'), 'flowdetails', ['parent_uuid'], unique=False)
```

Comment:
Shouldn't there be a corresponding `index=True` on the model definitions so doing a `create_all` for a fresh db accomplishes the same change?